src/OrchardCore.Modules/OrchardCore.Media/Assets.json

```diff
@@ -12,6 +12,7 @@
       "./node_modules/blueimp-file-upload/js/jquery.fileupload.js",
       "./node_modules/blueimp-file-upload/js/jquery.iframe-transport.js",
       "Assets/js/helpers.js",
+      "Assets/js/fontawesome-icons-thumbnails/**/*.js",
```

Comment:
I don't think you need a new folder for placing this file. You can place it in the MediaApp folder instead.